share/spice/clojure_packages_from_clojars/clojure_packages_from_clojars.js

```diff
@@ -0,0 +1,51 @@
+(function(env) {
+    "use strict";
+    env.ddg_spice_clojure_packages_from_clojars = function(api_result) {
+
+        // Validate the response (customize for your Spice)
+        var search_terms = DDG.get_query().split(" ");
+
+        function keep_it(candidate) {
```

Comment:
take a look at using a `relevancy: {}` block in the docs. It'll handle all of this for you :+1: